src/extension.ts

```diff
@@ -233,6 +233,8 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
         await launcher.runCurrentTarget();
     }));
 
+
+    /** Start of commands that shouldn't be exposed in package.json */
```

Comment:
Since you added this comment, you can also specify besides "shouldn't be exposed in package.json" that these are command substitution (used by VSCode in launch.json and tasks.json).

Reply:
Good idea. I'll add.